packages/components/src/input-control/types.ts

```diff
@@ -66,7 +73,8 @@ export type InputChangeCallback< P = {} > = (
 	extra: { event: SyntheticEvent } & P
 ) => void;
 
-export interface InputFieldProps extends BaseProps {
+export interface InputFieldProps
+	extends Omit< BaseProps, '__next36pxDefaultSize' > {
```

Comment:
After the props run through the `useDeprecated36pxDefaultSizeProp()` filter at the root of the component, no more `__next36pxDefaultSize` props should be trickling down to the subcomponents like `InputField`.